bin/rename_fasta.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python3
+
+'''Rename contigs of a FASTA file with incremental count.'''
+
+import argparse
+
+
+def main():
+    '''Execute renaming.'''
+
+    # Parse arguments.
+    parser = argparse.ArgumentParser(description='Rename FASTA files.', epilog='Work out those contigs.')
+    parser.add_argument('-i', '--input', help='indicate input FASTA file', required=True)
+    parser.add_argument('--pre', help='string pre contig count', type=str, default='')
```

Comment:
I think it is more consistent if all arguments have a short and a long version. 
```suggestion
    parser.add_argument('-p', '--pre', help='string pre contig count', type=str, default='')
```